src/main/java/uk/gov/hmcts/reform/prl/controllers/EditAndApproveDraftOrderController.java

```diff
@@ -226,6 +240,17 @@ public AboutToStartOrSubmitCallbackResponse populateCommonFields(
                 CaseData.class
             );
             Map<String, Object> response = draftAnOrderService.populateCommonDraftOrderFields(authorisation, caseData);
+            CreateSelectOrderOptionsEnum selectedOrder = (CreateSelectOrderOptionsEnum) response.get("orderType");
+            List<String> errorList = new ArrayList<>();
+            String dateOrderMade = response.get("dateOrderMade") != null
+                    ? response.get("dateOrderMade").toString() : " ";
```

Comment:
This needs discussion not sure why this condition is added.

Reply:
Have removed this